packages/ledger-icrc/src/index-ng.canister.spec.ts

```diff
@@ -167,4 +168,59 @@ describe("Index canister", () => {
       expect(res).toEqual(mockStatus);
     });
   });
+
+  describe("listSubaccounts", () => {
+    const mockSubaccounts: SubAccount[] = [
+      new Uint8Array([1, 2, 3, 4]),
+      new Uint8Array([5, 6, 7, 8]),
+    ];
+
+    it("should return the list of subaccounts for a Principal", async () => {
```

Comment:
"for an owner" instead of "for a principal"?

(a bit similar to my other comment, so generally speaking)

Reply:
Thanks for the feedback. Out of curiosity, is it a matter of wording, or is there something more that I'm not aware of?

Reply:
An ICRC account is defined (and standardized) by an `owner` and an optional `subaccount`. While the owner is technically of type `Principal`, it's definition is being an "owner", the "owner of the account".

Principal is basically a term that replaces a generic term "public key", so not tight to the notion of account and can be used in various context.

With that in mind, that was the reason why I suggest renaming the test from "list subaccounts from some Principal" to "list the subaccounts for an owner". I believe this phrasing is more meaningful from a business or use-case perspective related to the feature "listing subaccounts".

That said, it's just a suggestion, not a strong opinion.

Reply:
Makes sense?

Reply:
Yes, it does. Thanks for the explanation.